docs/framework/configure-apps/file-schema/wcf/httpstransport.md

```diff
@@ -24,12 +24,11 @@ Specifies an HTTP transport for transmitting SOAP messages for a custom binding.
                 maxBufferSize="Integer"
                 maxReceivedMessageSize="Integer"
                 proxyAddress="Uri"
-                proxyAuthenticationScheme="None/Digest/Negotiate/Ntlm/Basic/Anonymous"
+                proxyAuthenticationScheme="None/Digest/Negotiate/Ntlm/Basic/Anonymous/IntegratedWindowsAuthentication"
```

Comment:
https://github.com/dotnet/docs/blob/master/docs/framework/configure-apps/file-schema/wcf/httpstransport.md#attributes-and-elements

Reply:
I'm pretty certain this is incorrect. If you look at [this code ](https://referencesource.microsoft.com/#System.ServiceModel/System/ServiceModel/HttpProxyCredentialType.cs,29)you can see that it will throw an exception if anything other than Anonymous, Basic, Digest, Ntlm or Negotiate is specified.